fbfmaproom/fbfmaproom-sample.yaml

```diff
@@ -1934,6 +1945,27 @@ countries:
                     colormap: precip
                     lower_is_worse: yes
                     format: number0
+                bad-years-rank-v2:
+                    label: Reported bad years v2
+                    description: Adjusted rank from NMA’s Bad Year Classification (AA Workshop 2024)
```

Comment:
The apostrophe in this line is a non-ASCII unicode character, which caused the application to fail in the production container environment for some reason. We ought to be able to handle unicode strings, but while I look into that, please replace it with an ASCII apostrophe.

Reply:
replaced, from NMA’s to NMA's